CONTRIBUTING.md

```diff
@@ -138,11 +142,8 @@ If you are sending a pull request, please note the following:
   your problem or idea, which you want to solve with your
   pull request. We want to discuss the functionality within the issue and
   not guess the intention from the pull request!
-* In your pull request refer to the issue.
-* Describe your changes in the commit message.
-* At the beginning of the file, add the date, your name,
```

Comment:
i want to drop just this sentence "At the beginning of the file, add the date, your name,..." because it is done by git anyway, we have not done this consequently over time and it is just unnecessary.

Reply:
It is not done with Git in this file.
And sometimes during support I only saw one single file, because s.o. somehow "found s.th. somewhere." and copied sources.
Also think of our last discussion about licenses and copyright. You would not have the author of the piece directly within the piece it self. 
You could simply copy one file from the git repo and drop all git history.
You are right, it was not done consequently in the past. But this does not mean, it is right this way.

It was often left out by "core developers", who touched this file for the hundreths time. 
But imagine you have a file (without) git history, you would not know who is the author of this file. This is bad from a licence point of view!

(Also you can not argue, that the license itself is contained in the git repo, it has to be in the file).

Reply:
yes, i just mean to have the edit history in the file is obsolete with version control. If you edit the file in a meaningful way, you can add yourself to the authors in the license.

Reply:
I think we lately had a very good discussion with plettich and we will do it this way.

Reply:
@nilsbehlen  I would somehow keep these two:

* In your pull request refer to the issue.
* Describe your changes in the commit message.

We want readable commit messages. And referring to the issue helps reviewing the PR.

Reply:
Those 2 sentences are kept, i just added another half sentence to that.